src/test/integration/assessment/originalDataPoint.ts

```diff
@@ -47,7 +50,9 @@ export default (): void =>
 
     it('Edit existing/not existing Original data point', async () => {
       const editedOriginalDataPoint = await CycleDataController.updateOriginalDataPoint({
-        assessment,
+        assessment: {
+          ...assessment,
+        },
```

Comment:
why just spreading itself ? is it needed ?

Reply:
typo